wasm/section.go

```diff
@@ -132,7 +132,8 @@ func (m *Module) readSection(r *readpos.ReadPos) (bool, error) {
 	s.Start = r.CurPos
 
 	sectionBytes := new(bytes.Buffer)
-	sectionBytes.Grow(int(payloadDataLen))
+
+	sectionBytes.Grow(int(getInitialCap(payloadDataLen)))
```

Comment:
wouldn't it be clearer to return an error if `payloadDataLen` is larger than some max-value?
_ie:_ after line 128 above:
```go
if payloadDataLen > maxInitialCap {
    return errMaxAlloc // or something
}
```
```

Reply:
unfortunately, the wasm spec does not limit that data length. And it is hard to choose a reasonable constant max value. 

- If we set it too large, it still will cause memory risk, if the attacker can send a lot of wasm files concurrently;
- If we set it too small, it may limit some use cases;
- If we left it as param to let user fill in, it is a little cumbersome;

but this pr can handle it cleanly.